lib/Bloc/max_bloc.dart

```diff
@@ -52,9 +52,9 @@ Future<Max> getOneRepMax(String exercise) async {
         .where('exercise', isEqualTo: exercise)
         .orderBy('weight', descending: true)
         .get();
-  print(ref.docs.first.data());
+//   print(ref.docs.first.data());

```

Comment:
@0RaMsY0 I think we should remove lines that are unnecessary which are more often the commented ones.

Reply:
yeah i used that print() to determinat the content of the ref but now it should be removed